### PR TITLE
Add more samples, improvements

### PR DESCRIPTION
**Pull request by nirmal070125**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: hl7v2_to_json/main.bal [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+public function main() returns error? {
+    // Parse the HL7 message and ensure that it is of type ADT_A01.
+    hl7v23:ADT_A01 adtMsg = check hl7:parse(msg).ensureType(hl7v23:ADT_A01);
+    // transform the ADT_AO1 message to a custom patient record.
+    Patient patient = adta01ToPatient(adtMsg);
+    io:println("Custom patient json: ", patient.toJsonString());
+}
+
+function adta01ToPatient(hl7v23:ADT_A01 adtA01) returns Patient => {
+    firstName: msg,
```
Comment (Thiyanwso2) [inline hl7v2_to_json/main.bal:31]: Why are we setting the whole hl7 msg as the first name?

File: hl7v2_to_fhir/.devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "image": "ballerina/ballerina-devcontainer:2201.4.2",
```
Comment (Thiyanwso2) [inline hl7v2_to_fhir/.devcontainer.json:2]: I think Choreo is supporting version 5, so we can update the version here?

**Review by Thiyanwso2 — CHANGES_REQUESTED** [review]

File: working_with_fhir_connector/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# FHIR Connector Sample
+This sample demonstrate how to use pre-built FHIR connector to connect to
```
Comment (Thiyanwso2) [inline working_with_fhir_connector/README.md:2]: This sample demonstrate**s** how to use **the** pre-built FHIR connector to connect to Cerner EMR.

**Review by Thiyanwso2 — CHANGES_REQUESTED** [review]

File: hl7v2_to_json/main.bal (hunk 1 above)
Comment (nirmal070125) [reply hl7v2_to_json/main.bal:31]: Oh - mistake. thanks.

File: hl7v2_to_fhir/.devcontainer.json (hunk 2 above)
Comment (nirmal070125) [reply hl7v2_to_fhir/.devcontainer.json:2]: this is auto-gen. Let me remove this file.

File: hl7v2_to_fhir/main.bal [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import ballerina/io;
```
[the remark this answers is not in the document]
Comment (nirmal070125) [reply hl7v2_to_fhir/main.bal:1]: These are samples. It is not required IMO.

**Review by Thiyanwso2 — APPROVED** [review]

**Pull request merged**